data-mongodb/src/main/java/io/micronaut/data/mongodb/conf/MongoDataConfiguration.java

```diff
@@ -18,19 +18,17 @@
 import io.micronaut.context.annotation.ConfigurationProperties;
 import io.micronaut.context.condition.Condition;
 import io.micronaut.context.condition.ConditionContext;
+import io.micronaut.core.annotation.ReflectiveAccess;
 
 /**
  * Micronaut Data MongoDB integration configuration.
  */
 @ConfigurationProperties(MongoDataConfiguration.PREFIX)
+@ReflectiveAccess
 public final class MongoDataConfiguration {
 
     public static final String PREFIX = "micronaut.data.mongodb";
     public static final String CREATE_COLLECTIONS_PROPERTY = PREFIX + ".create-collections";
-    public static final String DRIVER_TYPE_PROPERTY = PREFIX + ".driver-type";
```

Comment:
@radovanradic why did you remove these?This would be a breaking change.

Reply:
I saw these were unused in the code, but forgot could be used elsewhere. Will revert